cmd/metal-api/internal/service/machine-service.go

```diff
@@ -938,7 +939,13 @@ func (r machineResource) allocateMachine(request *restful.Request, response *res
 		}
 	}
 
+	user, err := r.userGetter.User(request.Request)
+	if checkError(request, response, utils.CurrentFuncName(), err) {
+		return
+	}
+
 	spec := machineAllocationSpec{
+		Creator:            user.EMail,
```

Comment:
The firewall service also creates such an allocation spec. The value has to be passed there as well.

Also, there is a function `validateAllocationSpec` which you can use to ensure that this field was properly passed through to the machine allocation logic.

Reply:
done